### PR TITLE
Bug#1558417 - removed GitHub link and updated ansible variables

### PR DESCRIPTION
**Pull request by gaurav-nelson**
For https://bugzilla.redhat.com/show_bug.cgi?id=1558417

### CODE REVIEW

File: install_config/provisioners.adoc [hunk 1]
```diff
@@ -22,7 +22,6 @@ independently of {product-title}.
 
 [[provisioners-before-you-begin]]
 == Before You Begin
-An link:https://github.com/openshift/openshift-ansible/blob/master/playbooks/openshift-provisioners/config.yml[Ansible Playbook] is also available to deploy and upgrade external provisioners.
```
Comment (gaurav-nelson) [inline install_config/provisioners.adoc]: @gpei Can we keep this link in OpenShift origin?

File: install_config/provisioners.adoc [hunk 2]
```diff
@@ -55,21 +54,24 @@ which the `install` variable is `true`.
...
+|The prefix for the component images.
 ifdef::openshift-origin[]
-`openshift/origin-efs-provisioner:v1.0.0`, set prefix `openshift/origin-`.
+For example, with `openshift/origin-efs-provisioner:v1.0.0`, set prefix
+`openshift/origin-`.
 endif::[]
 ifdef::openshift-enterprise[]
-`openshift3/efs-provisioner:v3.6`, set prefix `openshift3/`.
+Defaults to `registry.access.redhat.com/openshift3/`, set it to a different
+value if you are using an alternative registry.
```
Comment (gaurav-nelson) [inline install_config/provisioners.adoc]: I have updated this for _enterprise_, are there any changes for _origin_?

File: install_config/provisioners.adoc [hunk 3]
```diff
@@ -55,21 +54,24 @@ which the `install` variable is `true`.
...
+|The version for the component images.
 ifdef::openshift-origin[]
-`openshift/origin-efs-provisioner:v1.0.0`, set version  as `v1.0.0`.
+For example, with `openshift/origin-efs-provisioner:v1.0.0`, set version  as
+`v1.0.0`.
 endif::[]
 ifdef::openshift-enterprise[]
-`openshift3/efs-provisioner:v3.6`, set version as `v3.6`.
+Defaults to `latest`.
 endif::[]
```
Comment (gaurav-nelson) [inline install_config/provisioners.adoc]: I have updated this for _enterprise_, are there any changes for _origin_?

**Comment by gaurav-nelson** [comment]
@gpei Are these changes only applicable to v 3.9? what about 3.5, 3.6, and 3.7?

File: install_config/provisioners.adoc (hunk 1 above)
Comment (gpei) [reply install_config/provisioners.adoc]: Not sure about this, we didn't use or test Origin.

File: install_config/provisioners.adoc (hunk 2 above)
Comment (gpei) [reply install_config/provisioners.adoc]: Sorry, have no idea about Origin

Comment (gaurav-nelson) [reply install_config/provisioners.adoc]: Thank you @gpei Are these changes only applicable to v 3.9? what about 3.5, 3.6, and 3.7?

**Comment by gpei** [comment]
hi @gaurav-nelson For the old versions, I just checked the openshift-ansible code and previous released official doc, 3.5 should have no such feature to be documented.

These changes could also be applicable to 3.6 & 3.7 after  default value of "openshift_provisioners_image_version" changed to `latest` in 3.6 & 3.7 as I requested in https://github.com/openshift/openshift-ansible/pull/7579.

One obvious difference between 3.9 & 3.6/3.7 is the path of playbook we used.
In 3.9, it's `/usr/share/ansible/openshift-ansible/playbooks/openshift-provisioners/config.yml`
In 3.6/3.7, it's `/usr/share/ansible/openshift-ansible/playbooks/byo/openshift-cluster/openshift-provisioners.yml`

BTW, in https://github.com/openshift/openshift-docs/pull/8289/files#diff-4ea3b85ecfd6ec8698eb3fc7cfd03f7aR213
just noticed there's another change needed.
$ ansible-playbook <OPENSHIFT_ANSIBLE_DIR>/playbooks/openshift-provisioners/config.yml \
    -e openshift_provisioners_install_provisioners=False
Should be:
$  ansible-playbook -v -i <inventory_file> \
     /usr/share/ansible/openshift-ansible/playbooks/openshift-provisioners/config.yml \
    -e openshift_provisioners_install_provisioners=False

File: install_config/provisioners.adoc [hunk 4]
```diff
@@ -153,7 +155,8 @@ The following command sets the directory in the EFS volume to
 be mountable and writeable by the provisioner pod.
 
 ----
-$ ansible-playbook <OPENSHIFT_ANSIBLE_DIR>/playbooks/openshift-provisioners/config.yml \
+$ ansible-playbook -v -i <inventory_file> \
+    /usr/share/ansible/openshift-ansible/playbooks/byo/openshift-cluster/openshift-provisioners.yml \
```
Comment (gaurav-nelson) [inline install_config/provisioners.adoc]: @gpei Thank you for reviewing this. I have made changes in this one for 3.6 and 3.7. After I merge this, I will create another PR for 3.9 specific changes.

**Pull request merged**